autogen/agentchat/contrib/gpt_assistant_agent.py

```diff
@@ -472,3 +477,31 @@ def find_matching_assistant(self, candidate_assistants, instructions, tools, fil
             matching_assistants.append(assistant)
 
         return matching_assistants
+
+    def process_assistant_config(self, llm_config, assistant_config):
```

Comment:
Suggestion: make the method internal by adding _

Reply:
Done